$NetBSD: patch-configure,v 1.1 2016/04/07 09:38:43 adam Exp $

Do not fail for missing LaTeX - it is not needed anyway.
Do not force optimisation flags.
Fix OpenMP detection.

--- configure.orig	2016-03-28 14:51:58.000000000 +0000
+++ configure
@@ -8716,10 +8716,6 @@ done
 test -n "$latex" || latex="no"
 
 export latex;
-if test $latex = "no" ;
-then
-        as_fn_error $? "Unable to find a LaTeX application" "$LINENO" 5;
-fi
 
 
 
@@ -11710,7 +11706,7 @@ if test "$enable_debug" = yes; then
 $as_echo "yes" >&6; }
     enable_debug=yes
 else
-    CXXFLAGS="$CXXFLAGS -O2 -DNDEBUG"
+    CXXFLAGS="$CXXFLAGS -DNDEBUG"
     { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
 $as_echo "no" >&6; }
     enable_debug=no
@@ -11763,7 +11759,7 @@ else
 ax_cv_cxx_openmp=unknown
 # Flags to try:  -fopenmp (gcc), -openmp (icc), -mp (SGI & PGI),
 #                -xopenmp (Sun), -omp (Tru64), -qsmp=omp (AIX), none
-ax_openmp_flags="-fopenmp -openmp -mp -xopenmp -omp -qsmp=omp none"
+ax_openmp_flags="-fopenmp -fopenmp=libiomp5 -openmp -mp -xopenmp -omp -qsmp=omp none"
 if test "x$OPENMP_CXXFLAGS" != x; then
   ax_openmp_flags="$OPENMP_CXXFLAGS $ax_openmp_flags"
 fi
@@ -11781,11 +11777,11 @@ for ax_openmp_flag in $ax_openmp_flags; 
 #ifdef __cplusplus
 extern "C"
 #endif
-char omp_set_num_threads ();
+#include <omp.h>
 int
 main ()
 {
-return omp_set_num_threads ();
+return omp_get_num_threads ();
   ;
   return 0;
 }
